cmd/main.go

```diff
@@ -453,6 +453,13 @@ func checkExistingAWSProfile(profileName string, config *vault.Config) error {
 	return nil
 }
 
+func getPartition(region string) string {
+	if partition, ok := endpoints.PartitionForRegion(endpoints.DefaultPartitions(), region); ok {
+		return partition.ID()
+	}
+	return "aws"
+}
```

Comment:
Why not always return partition.ID? Won't this always be either "aws" or "aws-gov"? If it errors out, isn't that a bad sign? I suspect this is a "Chas doesn't understand Go" problem, but I'm curious nevertheless.